action.yaml

```
@@ -47,6 +47,12 @@ inputs:
     description: Overwrite the function if there is one with the same name.
     default: false
     required: false
+  cpu:
+    description: Set number of CPU cores per function.
```

I would like us to somehow show/explain that this does not need to be a whole number / integer. Not sure what the best way is. One possibility might be:
```suggestion
  cpu:
    description: Set number of CPU cores per function, e.g. 0.8.
```